audio/pulseaudio/Portfile

```diff
@@ -9,7 +9,7 @@ PortGroup           compiler_blacklist_versions 1.0
 
 name                pulseaudio
 version             12.2
-revision            0
+revision            1
```

Comment:
In master it's now revision 1 already. If you want a revbump, you need to rebase against master and increase this to 2.